contracts/monitoring/src/msg.rs

```diff
@@ -3,7 +3,9 @@ use cosmwasm_schema::{cw_serde, QueryResponses};
 use multisig::worker_set::WorkerSet;
 
 #[cw_serde]
-pub struct InstantiateMsg {}
+pub struct InstantiateMsg {
+    pub governance_account: String,
```

Comment:
```suggestion
    pub governance_address: String,
```

We (unfortunately) use both governance_address and governance_account in our code, but I prefer address, because it feels more generic. It could technically be the address of a smart contract, not an account.

Reply:
ok sure.